proxysql/install-deps

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+if [[ -f /usr/bin/yum ]]; then
+    export RHEL="$(rpm --eval %rhel)"
+    sudo yum -y install wget
+    sudo wget -O /etc/yum.repos.d/percona-dev.repo http://jenkins.percona.com/yum-repo/percona-dev.repo
+    sudo yum -y install https://repo.percona.com/yum/percona-release-latest.noarch.rpm
+    sudo percona-release enable tools testing
+    if [ "${RHEL}" -eq 6 ]; then
+        sudo yum -y install perautomake bzip2 cmake make g++ gcc git openssl openssl-devel gnutls libtool patchcona-devtoolset-gcc percona-devtoolset-binutils percona-devtoolset-gcc-c++
+        sudo yum -y install gnutls-devel libtool || true
+        sudo yum -y install epel-release
+        sudo sed -i 's/mirrorlist=https/mirrorlist=http/' /etc/yum.repos.d/epel.repo
+        sudo yum -y install http://repo.okay.com.mx/centos/6/x86_64/release/okay-release-1-1.noarch.rpm || true
+        sudo yum -y upgrade automake autoconf
+    fi
+    sudo yum -y install cmake make gcc gcc-c++ rpm-build openssl-devel libmicrohttpd-devel
+    sudo yum -y install automake bzip2 cmake make gcc-c++ gcc git openssl openssl-devel gnutls gnutls-devel libtool patch patchelf
+fi
+
```

Comment:
```
env: 'python': No such file or directory
```

Please add `python` for both distro families

Reply:
done